domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/induction/dto/UpdateInductionScheduleStatusDto.kt

```diff
@@ -10,4 +10,5 @@ data class UpdateInductionScheduleStatusDto(
   val exemptionReason: String? = null,
   val latestDeadlineDate: LocalDate? = null,
   val prisonNumber: String,
+  val updatedAtPrison: String,
```

Comment:
Yes! 👍 
I noticed a minute ago (as part of the changes I'm making in my PR) that this was missing, so was going to mention it to you, but you've done it anyway 👍 😁